DEVELOPMENT.md

```diff
@@ -16,6 +16,9 @@ interactions, and other internal behavior, which leads to fragile tests.
 They also make it possible to mock complex interfaces or classes, which hides the fact that these classes are not (easily) testable. 
 We prefer to write mocks by hand, which forces code to be written in a certain testable style.
 * Use an assertion from Airlift's `Assertions` class if there is one that covers your case rather than writing the assertion by hand. Over time we may move over to more fluent assertions like AssertJ.
+* Using ``var`` is allowed.
```

Comment:
be more specific maybe.
```
Using ``var`` is allowed when it improves code readability and does not obscure types when reading code from GitHub
```
However, I thought we are more keen on not using ``vars`` as it's not clearly obvious when they improve readability

Reply:
My bad. Once again I forgot `not`.

Reply:
I disagree with a blanket ban on `var`. There are situation where it can make the code more readable. E.g., it can help break up long stream() chains and allow assigning meaningful names to intermediate steps without having to introduce convoluted type declarations that don't provide any value.

Reply:
would "is discouraged" better reflect the spirit of the slack conversations?

Reply:
I am fine with this. However, making cases white and black makes it easier to understand. Now, people might start to think about what kind of grey is allowed. To me it is wasted effort.

@martint Should I add you case as allowed exception?